custom/icds_reports/utils/aggregation_helpers/distributed/bihar_api_demographics.py

```diff
@@ -109,7 +109,8 @@ def aggregation_query(self):
             ('date_last_private_admit', 'person_list.date_last_private_admit '),
             ('date_return_private', 'person_list.date_return_private'),
             ('out_of_school_status', 'person_list.is_oos'),
-            ('last_class_attended_ever', 'person_list.last_class_attended_ever')
+            ('last_class_attended_ever', 'person_list.last_class_attended_ever'),
+            ('last_reported_fever_date', 'person_list.last_reported_fever_date')
```

Comment:
This field is added in another PR yeah? which means that should be merged before yeah

Reply:
It's merged https://github.com/dimagi/commcare-hq/pull/27452